CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with brackets that contains a dot at the end of them.
 * Fixed an issue where the **validate -g** failed reading a `.pack-ignore` file that contained only newlines and spaces.
 * Demisto-SDK will now exit gracefully with an appropriate error message when *git* is not installed.
+* Updated validation *RN116* to support the structure of **--force** flag in *update-release-notes* command.
```

Comment:
let's add that we fixed an issue that the rn was not added to git and that we fixed the structure of the rn when ran with -f